webpack.common.js

```diff
@@ -30,5 +31,13 @@ module.exports = {
       template: './src/client/template.ejs',
     }),
     new Dotenv(),
+    new styleLintPlugin({
+      configFile: '.stylelintrc',
+      context: 'src/client/styles',
+      files: '**/*.scss',
+      failOnError: false,
+      quiet: false,
+      syntax: 'scss',
+    }),
```

Comment:
Am I correct in understanding that the purpose of this plugin is just to alert you if you've goofed while developing? Should it be in `webpack.dev.js` instead? Not sure if style errors should cause build failures.